shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -564,6 +565,10 @@ - (UIView*)splashScreenView {
   return _splashScreenView.get();
 }
 
+- (VSyncClient*)keyboardAnimationVsyncClient {
+  return _keyboardAnimationVsyncClient.get();
+}
```

Comment:
Can prob remove this method if we make `keyboardAnimationVsyncClient` a property.

Reply:
Done